src/pocketmine/tile/Sign.php

```diff
@@ -64,6 +64,33 @@ public function setText($line1 = "", $line2 = "", $line3 = "", $line4 = ""){
 
 		return true;
 	}
+	
+	/**
+	 * @param int    $index 0-3
+	 * @param string $line
+	 * @param bool   $update
+	 */
+	public function setLine(int $index, string $line, bool $update = true){
```

Comment:
well, if you want to change multiple lines you don't need to update every time. It is not necessary.